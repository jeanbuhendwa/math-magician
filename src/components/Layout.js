import { Outlet } from 'react-router-dom';
import Navbar from './NavBar';

const Layout = () => (
  <div>
    <Navbar />
    <Outlet />
  </div>
);

export default Layout;
