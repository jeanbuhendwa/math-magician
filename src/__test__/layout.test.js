import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Layout from '../components/Layout';

describe('Layout', () => {
  it('renders correctly', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <Layout />
      </MemoryRouter>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
