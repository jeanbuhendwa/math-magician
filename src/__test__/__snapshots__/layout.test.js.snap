// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<DocumentFragment>
  <div>
    <nav
      class="navBar"
    >
      <p
        class="logo"
      >
        Math
        <span
          class="logoSpan"
        >
          Magician
        </span>
      </p>
      <ul>
        <li>
          <a
            aria-current="page"
            class="active"
            href="/"
          >
            Home
          </a>
        </li>
        <li>
          <a
            class=""
            href="/calculator"
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            class=""
            href="/quote"
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </div>
</DocumentFragment>
`;
